src/git-review.zig

```diff
@@ -389,8 +429,46 @@ const Review = struct {
             try writer.print("{d} resolved\n", .{review.resolved_count});
         }
     }
+
+    pub fn format_email(
+        comments: []const Comment,
+        writer: std.io.AnyWriter,
+    ) !void {
+        try writer.writeAll("New review comments:\n");
+
+        var delimiter: []const u8 = "";
+        for (comments) |comment| {
+            try writer.writeAll(delimiter);
+            delimiter = "\n";
+            try writer.print("{}:\n", .{comment.location});
+            try writer.writeAll(comment.text);
+        }
+    }
 };
 
+// Every program expands until it can send email...
+fn send_email(shell: *Shell, subject: []const u8, body: []const u8, options: EmailOptions) !void {
+    assert(stdx.cut(subject, "\n") == null);
+    const email = try shell.fmt(
+        \\From: "Git Review" <{s}>
+        \\To: <{s}>
+        \\Subject: {s}
+        \\
+        \\{s}
+    , .{ options.sender, options.recepient, subject, body });
+
+    try shell.exec_options(.{ .stdin_slice = email },
+        \\curl --ssl-reqd {server} --user {auth}
+        \\  --mail-from {sender} --mail-rcpt {recepient}
+        \\  --upload-file -
+    , .{
+        .server = options.server,
+        .auth = options.auth,
+        .sender = options.sender,
+        .recepient = options.recepient,
+    });
```

Comment:
Open to suggestions for how to send email better, but just curling seems the optimal solution to me. This works with a gmail account using https://myaccount.google.com/apppasswords